apl-desktop/src/main/java/com/apollocurrency/aplwallet/apldesktop/DesktopApplication.java

```diff
@@ -82,7 +76,7 @@
     private static final DbRecoveringUI DB_RECOVERING_UI = DbRecoveringUI.getInstance();
     private static final boolean ENABLE_JAVASCRIPT_DEBUGGER = false;
     private static volatile boolean isSplashScreenLaunched = false;
-    private static volatile Stage mainStage;
+    static volatile Stage mainStage;
```

Comment:
I didn't get how instance is initialized? Can you explain ?

Reply:
com.apollocurrency.aplwallet.apldesktop.DesktopApplication#start